google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -185,22 +192,92 @@ public static TransportChannelProvider defaultTransportChannelProvider() {
         .setJwtEnabledScopes(JWT_ENABLED_SCOPES);
   }
 
-  /** Returns the object with the settings used for calls to ReadRows. */
+  /**
+   * Returns the object with the settings used for calls to ReadRows.
+   *
+   * <p>This is idempotent and streaming operation.
+   *
+   * <p>The idle timeout is configured via {@code ServerStreamingCallSettings#getIdleTimeout}. This
+   * timer will terminate any stream that has not has seen any demand or activity in the configured
+   * interval. To turn off idle checks, set the idleTimeout to {@link Duration#ZERO}.
```

Comment:
Please document the default value here (5 minutes)

Reply:
I have rephrased this line with default time.